app/assets/javascripts/components/_navbar.js.jsx

```diff
@@ -0,0 +1,16 @@
+var Navbar = React.createClass({
+  render() {
+    return (
+      <nav>
+       <div class="nav-wrapper">
+         <a href="/" class="brand-logo">MENTORSHIP</a>
+         <ul id="nav-mobile" class="right hide-on-med-and-down">
+           <li><a href="sass.html">Sass</a></li>
+           <li><a href="badges.html">Components</a></li>
+           <li><a href="collapsible.html">JavaScript</a></li>
+         </ul>
```

Comment:
So the navbar has the three links? one for sass, components and javascript? Dummy links?